src/component/CountryPage/CountryPage.jsx

```diff
@@ -0,0 +1,57 @@
+import { CardHeader, CardContent } from "@material-ui/core";
+import { Box, CardCountry, CardMedia, Stamp } from "../../style/style";
+
+const CountryPage = (props) => {
+  let card = props.countries.find((country) => {
+    return country.name === props.nameCountry;
+  });
+  if (!card) {
+    card = props.selectedCards.find((country) => {
+      return country.name === props.nameCountry;
+    });
+  }
+
+  if (card) {
+    localStorage.setItem("card", JSON.stringify(card));
+  }
+
+  localStorage.setItem(
```

Comment:
try to google `url search params`  you could store selected ID-s and search string from the input there